domain-models-runtime/src/test/java/org/folio/rest/persist/ddlgen/SchemaMakerIT.java

```diff
@@ -241,6 +241,11 @@ private void assertSchemaSelectSingle(TestContext context, String sql, String ex
     assertSelectSingle(context, sql, expected);
   }
 
+  @Test
+  public void f_unaccent_combining_character(TestContext context) {
+    assertSchemaSelectSingle(context, "SELECT f_unaccent(E'a\\u0308 and a\\u0308')", "a and a");
```

Comment:
More than just `\\u0308` could be tested here, but given that they're all part of the same regex expression it's probably not a huge deal.